### PR TITLE
Awais786/ecom 6041 email templates

### PR DESCRIPTION
**Pull request by awais786**
Update the email templates for studio instance creation.

https://openedx.atlassian.net/browse/ECOM-6041

### CODE REVIEW

**Comment by awais786** [comment]
@waheedahmed @tasawernawaz Please review.

File: course_discovery/apps/publisher/emails.py [hunk 1]
```diff
@@ -64,11 +65,24 @@ def send_email_for_studio_instance_created(course_run):
         to_addresses = course_run.course.get_course_users_emails()
         from_address = settings.PUBLISHER_FROM_EMAIL
 
+        course_user_roles = course_run.course.course_user_roles.all()
+        user_roles = {}
```
Comment (waheedahmed) [inline course_discovery/apps/publisher/emails.py]: Why making a dict when you can directly get user roles like this:
```
course_user_roles = course_run.course.course_user_roles.all()
course_team = course_user_roles.get(role=PublisherUserRole.CourseTeam)
partner_coordinator = course_user_roles.get(role=PublisherUserRole.PartnerCoordinator)
```

**Review by waheedahmed — APPROVED** [review]
Just one comment, 👍 once addressed

**Comment by awais786** [comment]
@bderusha Please review this PR.

File: course_discovery/templates/publisher/email/studio_instance_created.html [hunk 2]
```diff
@@ -2,11 +2,21 @@
 {% load i18n %}
 {% block body %}
 <!-- Message Body -->
+<p>{% trans "Dear"%} {{ course_team }},<p>
```
Comment (bderusha) [inline course_discovery/templates/publisher/email/studio_instance_created.html]: Please make this include the `{{ course_team }}` variable in the translation.  Also add a translators note so they know the context of this.

File: course_discovery/templates/publisher/email/studio_instance_created.html [hunk 3]
```diff
@@ -2,11 +2,21 @@
...
         <p>
             {% blocktrans with link_start='<a href="' link_middle='">' link_end='</a>' trimmed %}
-                Studio instance created for the following course run: {{ link_start }}{{ course_run_page_url }}{{ link_middle }}course run link{{ link_end }}.
+                EdX has created a Studio instance for {{ link_start }}{{ course_run_page_url }}{{ link_middle }} {{ course_name }} {{ link_end }}.
+                You can now edit this course in Studio.
             {% endblocktrans %}
         </p>
-        <p>{% trans "The edX team" %}</p>
+
+        <p>{% trans "Thanks"%},</p>
```
Comment (bderusha) [inline course_discovery/templates/publisher/email/studio_instance_created.html]: include the comma and add a translators comment noting that this is the closing of a letter.

File: course_discovery/templates/publisher/email/studio_instance_created.html (hunk 2 above)
Comment (bderusha) [reply course_discovery/templates/publisher/email/studio_instance_created.html]: here and .txt version

File: course_discovery/templates/publisher/email/studio_instance_created.html (hunk 3 above)
Comment (bderusha) [reply course_discovery/templates/publisher/email/studio_instance_created.html]: here and .txt version

**Review by bderusha — APPROVED** [review]
👍 after minor comments addressed

**Pull request merged**